assets/tasks/common.json

```diff
@@ -285,38 +260,6 @@
                 "group": "cloud-related"
             }
         },
-        {
-            "label": "docker-login",
```

Comment:
The same here.

Reply:
@andreriesco could you instead remove add a description and set the hide: false ?

Reply:
Sure